ethereum-consensus/src/electra/block_processing.rs

```diff
@@ -1,10 +1,230 @@
+use ssz_rs::{HashTreeRoot, List};
+
 use crate::{
-    electra::{Attestation, BeaconBlockBody, BeaconState},
+    crypto::fast_aggregate_verify,
+    electra::{
+        add_flag, compute_consolidation_epoch_and_update_churn, compute_domain,
+        compute_epoch_at_slot, compute_exit_epoch_and_update_churn, compute_signing_root,
+        compute_timestamp_at_slot, decrease_balance, get_attestation_participation_flag_indices,
+        get_attesting_indices, get_base_reward, get_beacon_committee, get_beacon_proposer_index,
+        get_committee_count_per_slot, get_committee_indices, get_consolidation_churn_limit,
+        get_current_epoch, get_indexed_attestation, get_pending_balance_to_withdraw,
+        get_previous_epoch, get_randao_mix, get_validator_max_effective_balance,
+        has_compounding_withdrawal_credential, has_eth1_withdrawal_credential,
+        has_execution_withdrawal_credential, has_flag, increase_balance, initiate_validator_exit,
+        invalid_operation_error, is_active_validator, is_compounding_withdrawal_credential,
+        is_fully_withdrawable_validator, is_partially_withdrawable_validator,
+        is_valid_indexed_attestation, kzg_commitment_to_versioned_hash,
+        mainnet::MIN_ACTIVATION_BALANCE, process_attester_slashing,
```

Comment:
I would think `MIN_ACTIVATION_BALANCE` is a `config` item, and so it would come from the `context`